src/components/core/Navigation.vue

```diff
@@ -119,6 +121,7 @@ function promptToEnablePlugin(pluginName) {
             .dropdown-menu(role="menu")
                 .dropdown-content.ml-2
                     router-link.dropdown-item(to="/agents") agents
+                    router-link.dropdown-item(to="/payloads") payloads
```

Comment:
You think Payloads are a better fit in the "Campaigns" menu versus the "Configuration" menu?

Curious what @clenk  thinks.

Reply:
Yea I think I prefer it under "Configuration", if thats okay